FirebaseInstallations/Source/Library/InstallationsIDController/FIRInstallationsIDController.m

```diff
@@ -158,9 +159,16 @@ - (instancetype)initWithGoogleAppID:(NSString *)appID
 }
 
 - (FBLPromise<FIRInstallationsItem *> *)createAndSaveFID {
-  return [self migrateOrGenerateFID].then(^FBLPromise<FIRInstallationsItem *> *(NSString *FID) {
-    return [self createAndSaveInstallationWithFID:FID];
-  });
+  return [self migrateOrGenerateFID]
+      .then(^FBLPromise<FIRInstallationsItem *> *(NSString *FID) {
+        return [self createAndSaveInstallationWithFID:FID];
+      })
+      .then(^FIRInstallationsItem *(FIRInstallationsItem *installation) {
+        [[NSNotificationCenter defaultCenter]
+            postNotificationName:FIRInstallationIDDidChangeNotification
```

Comment:
Do we want to pass along the item here or no?

Reply:
Good question. I would no pass entire item, because the intention is that the developer would re-request installation or auth token if needed. Also, at this point, the installation does not have an auth token yet. Though, we probably should pass info on which FIRApp the installation was updated for. 
I'll probably confirm it with Rayo and implement it separately.

Reply:
Makes sense, thanks!